specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/EdgeSubscription.json

```diff
@@ -0,0 +1,403 @@
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/EdgeSubscriptionList"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          }
+        }
+      },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.AzureStack/edgeSubscriptions/{edgeSubscriptionName}": {
+        "get": {
+          "x-ms-examples": {
+            "Returns the properties of an edge suscription.": {
+              "$ref": "examples/EdgeSubscription/GetEdgeSub.json"
+            }
+          },
+          "tags": [
+            "EdgeSubscriptions"
+          ],
+          "description": "Returns the properties of an Azure Stack edge subscriptions.",
+          "operationId": "EdgeSubscriptions_Get",
+          "parameters": [
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/ResourceGroupNameParameter"
+            },
+            {
+                "$ref": "#/parameters/EdgeSubscriptionNameParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/EdgeSubscription"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          }
+        },
+        "delete": {
+          "x-ms-examples": {
+            "Delete the requested Azure Stack edge subscription.": {
+              "$ref": "examples/EdgeSubscription/DeleteEdgeSub.json"
+            }
+          },
+          "tags": [
+            "EdgeSubscriptions"
+          ],
+          "description": "Delete the requested Azure Stack edge subscription.",
+          "operationId": "EdgeSubscriptions_Delete",
+          "parameters": [
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/ResourceGroupNameParameter"
+            },
+            {
+              "$ref": "#/parameters/EdgeSubscriptionNameParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK"
```

Comment:
same comment on response descriptions - how does 200 differ from 204 for this operation?

Reply:
Updated.